src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -111,222 +57,127 @@ export const TransactionGenericRow = ({
-    return Object.values(transfersByType).map((transfersOfType, index) => {
-      const hasManyTypeTransfers = transfersOfType.length > 1
-
+  const renderSendInfo = useMemo(() => {
+    if (transfersByType.Send && transfersByType.Send.length > 1) {
+      const assets = transfersByType.Send.map(transfer => transfer.asset.symbol)
+      return <RawText color='text.subtle'>{assets.join('/')}</RawText>
+    }
+    if (transfersByType.Send && transfersByType.Receive) {
+      const precision = transfersByType.Send[0].asset.precision ?? FALLBACK_PRECISION
+      const amount = fromBaseUnit(transfersByType.Send[0].value, precision)
       return (
-        <React.Fragment key={index}>
-          {hasManyTypeTransfers ? (
-            <AssetsTransfers index={index} compactMode={compactMode} transfers={transfersOfType} />
-          ) : (
-            <AssetTransfer index={index} compactMode={compactMode} transfer={transfersOfType[0]} />
-          )}
-        </React.Fragment>
+        <Amount.Crypto
+          color='text.subtle'
+          value={amount}
+          symbol={transfersByType.Send[0].asset.symbol}
+          maximumFractionDigits={4}
+          prefix='-'
+        />
       )
-    })
-  }, [compactMode, transfersByType])
-
-  const isNft = useMemo(() => {
-    return Object.values(transfersByType)
-      .flat()
-      .some(transfer => !!transfer.id)
+    }
   }, [transfersByType])
 
-  const cryptoValue = useMemo(() => {
-    if (!fee) return '0'
-    return fromBaseUnit(fee.value, fee.asset.precision)
-  }, [fee])
-
-  const fiatValue = useMemo(() => {
-    return bnOrZero(fee?.marketData?.price)
-      .times(cryptoValue)
-      .toString()
-  }, [fee?.marketData?.price, cryptoValue])
-
-  const gridTemplateColumns = useMemo(() => ({ base: '1fr', md: columns }), [columns])
-  const iconCircleBoxSize = useMemo(
-    () => ({ base: '24px', lg: compactMode ? '24px' : '38px' }),
-    [compactMode],
-  )
-  const stackDirection: StackDirection = useMemo(
-    () => ({ base: 'row', md: 'column', xl: compactMode ? 'row' : 'column' }),
-    [compactMode],
-  )
-  const stackFontSize = useMemo(
-    () => ({ base: 'sm', lg: compactMode ? 'sm' : 'md' }),
-    [compactMode],
-  )
-
-  const stackAlignItems = useMemo(
-    () => ({ base: 'flex-start', xl: compactMode ? 'center' : 'flex-start' }),
-    [compactMode],
-  )
-  const tagMinHeight = useMemo(
-    () => ({ base: '1.2rem', md: compactMode ? '1.2rem' : '1.25rem' }),
-    [compactMode],
-  )
-  const tagPx = useMemo(() => ({ base: 2, md: compactMode ? 2 : 2 }), [compactMode])
-  const stackSpacing = useMemo(() => ({ base: 0, md: 4, xl: compactMode ? 0 : 4 }), [compactMode])
-  const stackJustifyContent = useMemo(
-    () => ({
-      base: 'space-between',
-      md: 'flex-start',
-      xl: compactMode ? 'space-between' : 'flex-start',
-    }),
-    [compactMode],
-  )
+  const renderBody = useMemo(() => {
+    if (transfersByType.Receive && transfersByType.Receive.length > 1) {
```

Comment:
We may want to extract these common checks to vars same as e.g `hasManyTypeTransfers` previously